assets/eip-5139/AUTHORS.md

```diff
@@ -0,0 +1,37 @@
+SemVer Authors
```

Comment:
@SamWilsn Not sure this is meant to be included?

Reply:
Yep, it's to comply with the CC1.0-BY license.